nx-X11/programs/Xserver/os/oscolor.c

```diff
@@ -45,438 +44,1619 @@ SOFTWARE.
 
 ******************************************************************/
 
-#ifdef NX_TRANS_SOCKET
-
-#include <sys/types.h>
-#include <sys/stat.h>
-#include <unistd.h>
-
-/* FIXME: we need more intelligent code (like provided by the nxagentX2go
- * var in hw/nxagent/Init.h) to detect our current runtime mode (running
- * as x2goagent, running as nxagent)
- */
-static char* nxAltRgbPaths[] = {"/etc/x2go/rgb", \
-                                "/usr/share/x2go/rgb", \
-                                "/usr/local/share/x2go/rgb", \
-                                "/etc/nxagent/rgb", \
-                                "/usr/share/nx/rgb", \
-                                "/usr/local/share/nx/rgb", \
-                                "/usr/NX/share/rgb", \
-                                "/usr/share/X11/rgb", \
-                                "/etc/X11/rgb"};
-static char _NXRgbPath[1024];
-
-#endif
-
 #ifdef HAVE_DIX_CONFIG_H
 #include <dix-config.h>
 #endif
 
-#ifndef USE_RGB_TXT
-
-#ifdef NDBM
-#include <ndbm.h>
-#else
-#ifdef SVR4
-#include <rpcsvc/dbm.h>
-#else
-#include <dbm.h>
-#endif
-#endif
-#include "rgb.h"
+#include <X11/keysym.h>
```

Comment:
Why do we need that? Looks misplaced, keysyms shouldn't be used in `oscolor.c` at all.